File: 6KYU/Multiplication table/solution.cs
using System;

namespace Solution
{
  class Solution
  {
    public static int[,] MultiplicationTable(int size)
    {
      int [,] res = new int[size, size];
      for(var i = 0; i < size; i++)
         for(var j = 0; j < size; j++)
            res[i,j] = (i+1) * (j+1);
      
      return res;
    }
  }
}

File: 6KYU/Multiplication table/solution.go
package multiplicationtable

func MultiplicationTable(size int) [][]int {
	var res [][]int

	for i := 0; i < size; i++ {
		row := make([]int, size)
		res = append(res, row)
	}

	for i := 0; i < size; i++ {
		for j := 0; j < size; j++ {
			res[i][j] = (i + 1) * (j + 1)
		}
	}
	return res
}
